Source/Tools/FEXLoader/TestHarnessRunner.cpp

```diff
@@ -168,7 +255,15 @@ int main(int argc, char **argv, char **const envp) {
 
   CTX->InitializeContext();
 
+#ifndef _WIN32
   auto SignalDelegation = FEX::HLE::CreateSignalDelegator(CTX.get());
+#else
+  // Enable exit on HLT while Wine's longjump is broken.
+  //
+  // Once they fix longjump, we can remove this.
```

Comment:
Is there any reference to an upstream bug or similar to this? As is, the "Once they fix longjmp, we can remove this" comment isn't very useful since there's no way to tell if it's fixed unless you already knew about it.

Reply:
Nothing currently tracked in upstream wine about this issue.